### PR TITLE
Added env DOCKER_RUNNING=true in Dockerfile

### PR DESCRIPTION
**Pull request by WeiChungHsu**
To added env DOCKER_RUNNING=true in Dockerfile, and RUN commands could use this var to detect if it is inside the docker container.

### CODE REVIEW

**Review by 0405ysj — APPROVED** [review]

**Comment by ikicha** [comment]
Isn't checking if "/.dockerenv" exist enough?

**Comment by 0405ysj** [comment]
> Isn't checking if "/.dockerenv" exist enough?

In his experiment, `/.dockerenv` wasn't recognized in `cuttlefish-base.postinst`. That lead `kvm` group being missed.

**Comment by WeiChungHsu** [comment]
> > Isn't checking if "/.dockerenv" exist enough?
> 
> In his experiment, `/.dockerenv` wasn't recognized in `cuttlefish-base.postinst`. That lead `kvm` group being missed.

Thanks for helping me answer the question. And yes, the `/.dockerenv` is missing when running Dockerfile RUN commands. I search google, stockoverflow and docker sites, this file `/.dockerenv` is not persistent and a good way to check if you are inside docker.

On the other hand, there is no good way and no official suggested solution to detect if you are inside a docker.

File: docker/arm-server/Dockerfile [hunk 1]
```diff
@@ -3,6 +3,9 @@
 # so it could execute CF instance with API in HO.
 FROM debian:stable-20211011 AS cuttlefish-arm64
 
+# to detect if it is running inside a docker
+ENV DOCKER_RUNNING=true
```
Comment (jemoreira) [inline docker/arm-server/Dockerfile:7]: There is code in other places (example: cvd) that checks for the existence of the `.dockerenv` file. It's probably best to make sure it exists here than to come up with another mechanism, something like `RUN touch /.dockerenv` should work.

**Review by jemoreira — CHANGES_REQUESTED** [review]

**Review by jemoreira — APPROVED** [review]

**Pull request merged**